3_javascript/javascript1/week3/script.js

```diff
@@ -0,0 +1,218 @@
+//Java-scipt Week3 Homework//
+
+//(1) - Item array removal// 
+
+const names = [
+    "Peter",
+    "Ahmad",
+    "Yana",
+    "kristina",
+    "Rasmus",
+    "Samuel",
+    "katrine",
+    "Tala",
+  ];
+  const nameToRemove = "Ahmad";
+  const filteredNames = names.filter(name => name !== nameToRemove);
+  
+  console.log(filteredNames); // ['Peter', 'Yana', 'kristina', 'Rasmus', 'Samuel', 'katrine', 'Tala']
+  
+
+  //When will we be there??//
+
+  function calculateTravelTime(travelInformation) {
+    const { speed, destinationDistance } = travelInformation;
+    const totalTimeInHours = destinationDistance / speed;
+    const hours = Math.floor(totalTimeInHours);
+    const minutes = Math.round((totalTimeInHours - hours) * 60);
+  
+    const formattedTime = `${hours} hours and ${minutes} minutes`;
+  
+    return formattedTime;
+  }
+  
+  const travelInformation = {
+    speed: 50,
+    destinationDistance: 432,
+  };
+  
+  const travelTime = calculateTravelTime(travelInformation);
+  console.log(travelTime);
+
```

Comment:
🌟 The function `calculateTravelTime` is well-structured. I love how you've used destructuring from the input object to keep your code clean.
💡 Suggestion: Maybe provide a brief comment on top of the function to give a quick overview of its functionality.